modules/hal_silabs/gecko/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+menu "Gecko SDK configuration"
+	depends on HAS_SILABS_GECKO
+
+config SILABS_GECKO_RAIL_MULTIPROTOCOL
+	bool "Use RAIL Multiprotocol library"
+	depends on SOC_GECKO_USE_RAIL
+	default n
```

Comment:
Boolean kconfig options default to `n`, and shouldn't be explicitly set according to https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults